### ISSUE TITLE
Alpine DNS Issues

### ISSUE THREAD

**Issue by nhairs** [aSauerwein/parsedmarc-container#4]
Hi there,

I wanted to flag that Alpine might not be the best base image to use for this project due to the way the local resolver handles DNS lookups. Granted this only applies if parsedmarc is configured to use the local resolver rather than the default `1.1.1.1`.

For more detail see: https://martinheinz.dev/blog/92

**Comment by aSauerwein** [comment]
Hi!

thanks for pointing that out, didn't know of that issue with Alpine Linux and DNS. 
but for me it seems like it has been fixed in 3.19
```bash
/ # cat /etc/os-release 
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.19.0
PRETTY_NAME="Alpine Linux v3.19"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"
/ # dig +tcp @9.9.9.9 github.com

; <<>> DiG 9.18.19 <<>> +tcp @9.9.9.9 github.com
; (1 server found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 12473
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;github.com.			IN	A

;; ANSWER SECTION:
github.com.		27	IN	A	140.82.121.4

;; Query time: 28 msec
;; SERVER: 9.9.9.9#53(9.9.9.9) (TCP)
;; WHEN: Mon Jan 01 15:18:20 UTC 2024
;; MSG SIZE  rcvd: 55

/ # 
```

maybe even in 3.18: https://www.theregister.com/2023/05/16/alpine_linux_318/

**Comment by nhairs** [comment]
Ah that's good to know! Sorry for the incorrect report :sweat_smile:
